src/components/definition-list/definition-list.style.js

```diff
@@ -18,15 +18,15 @@ export const StyledDtDiv = styled.div`
   ${({ w, dtTextAlign }) => css`
     text-align: ${dtTextAlign};
     width: ${w}%;
-    white-space: nowrap;
+    white-space: normal;
```

Comment:
`normal` is the default value for `white-space`. Could we simply remove this rule?